src/pause.rs

```diff
@@ -0,0 +1,98 @@
+//! Contains functionality of pause and resume container command
```

Comment:
I would either split this up into two files (pause and resume) or rename the file to freeze.rs. Pause.rs does not reflect the complete content of the file.

Reply:
I chose the former.